src/features/languages/content/client/descriptions.json

```diff
@@ -737,6 +737,7 @@
     "config.form.settings.warning": "Success notification label for number of addresses settings",
     "config.form.settings.link": "Success notification label for number of addresses settings",
     "config.form.settings.fetchButton": "Success notification label for number of addresses settings",
+    "config.form.settings.nidVerificationButton": "Success notification label for number of addresses settings",
```

Comment:
Why are all of these "Success notification label for number of addresses settings"? Should it rather be something like "Success notification label for National ID verification button"?